framework/wazuh/tests/data/schema_syscollector_000.sql

```diff
@@ -183,6 +183,10 @@ INSERT INTO sys_osinfo VALUES (2011369001, '2019/03/21 10:25:00', 'agent', 'x86_
 INSERT INTO sys_hwinfo VALUES (2089525312, '2019/03/21 11:25:00', '0', 'Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz', 2,
                                1992.001, 492832, 64032, 88);
 
+/* accept empty/NULL values at cpu fields */
+INSERT INTO sys_hwinfo VALUES (2089525313, '2023/01/31 9:00:00', '0', 'Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz', 0,
+                               0.0, 0, 0, "0000");

```

Comment:
Question: why are we adding a string `"0000"` as `ram_usage INTEGER` ?

Reply:
It is an unfinished test, I will return to the original.